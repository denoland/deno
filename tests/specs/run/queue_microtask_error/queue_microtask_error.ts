queueMicrotask(() => {
  throw new Error("foo");
});
console.log(1);
Promise.resolve().then(() => console.log(2));
