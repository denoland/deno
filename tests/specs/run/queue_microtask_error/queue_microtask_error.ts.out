1
error: Uncaught Error: foo
  throw new Error("foo");
        ^
    at [WILDCARD]/queue_microtask_error.ts:2:9
    at ext:core/[WILDCARD]
